VGA_applicatie_groep1.launch

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
deze file had in de git ignore gemogen denk ik

Reply:
OK